snaps/features/custom-evm-accounts/security.md

```diff
@@ -177,12 +177,13 @@ For example:
   }
   ```
 
-## Expose Account Management API methods using the onKeyringRequest entry point
+## Expose Account Management API methods using the `onKeyringRequest` entry point
 
 The [`onRpcRequest`](../../reference/entry-points.md#onrpcrequest) entry point is a general-purpose
 entry point and has no restrictions on the methods that can be called.
-Ensure that you only expose Account Management API methods using the `onKeyringRequest` entry point, which has
-extra security checks.
+Ensure that you only expose Account Management API methods using the
+[`onKeyringRequest`](../../reference/entry-points.md#onkeyringrequest) entry point, which has extra
```

Comment:
Currently this brings the user to the top of the page, if we use this [link](https://github.com/MetaMask/metamask-docs/blob/dbfb78b25696b32de5ae7613d9f7d340431701b3/snaps/reference/entry-points.md#onkeyringrequest) if will take the user directly to the onKeyringRequest section